src/sinks/console.rs

```diff
@@ -124,9 +123,19 @@ struct WriterSink {
 impl StreamSink for WriterSink {
     async fn run(&mut self, mut input: BoxStream<'_, Event>) -> Result<(), ()> {
         while let Some(event) = input.next().await {
-            write_event_to_output(&mut self.output, event, &self.encoding)
+            let buf = match encode_event(event, &self.encoding) {
+                Some(mut buf) => {
+                    buf.push('\n');
+                    buf
+                }
+                None => continue,
```

Comment:
`self.acker.ack(1)` should be called anyway

Reply:
moved `ack` to the top.